apps/omg_watcher/lib/exit_processor/core.ex

```diff
@@ -492,10 +493,20 @@ defmodule OMG.Watcher.ExitProcessor.Core do
       late_invalid_exits
       |> Enum.map(fn {position, late_exit} -> ExitInfo.make_event_data(Event.UnchallengedExit, position, late_exit) end)
 
+    # get exits which are invalid because of being spent in IFEs
+    invalid_exits_based_on_ifes_events =
```

Comment:
now remove variable invalid_exits_based_on_ifes_events, because function ```get_invalid_exits_based_on_ifes``` tell about variable.
why invalid_exits_based_on_ifes_events don't return ```EventData```?

Reply:
I just went with convention in this function:
1. get data
2 prepare events

Reply:
I did:
```
  # get exits which are invalid because of being spent in IFEs
    invalid_exit_events =
      get_invalid_exits_based_on_ifes(state)
      |> Enum.map(fn {position, exit_info} -> ExitInfo.make_event_data(Event.InvalidExit, position, exit_info) end)
      |> Enum.concat(non_late_events)
      |> Enum.uniq_by(fn %Event.InvalidExit{utxo_pos: utxo_pos} -> utxo_pos end)
```